Source/Core/Core/Config/GraphicsSettings.cpp

```diff
@@ -148,6 +148,7 @@ const Info<bool> GFX_HACK_COPY_EFB_SCALED{{System::GFX, "Hacks", "EFBScaledCopy"
 const Info<bool> GFX_HACK_EFB_EMULATE_FORMAT_CHANGES{
     {System::GFX, "Hacks", "EFBEmulateFormatChanges"}, false};
 const Info<bool> GFX_HACK_VERTEX_ROUDING{{System::GFX, "Hacks", "VertexRounding"}, false};
+const Info<u8> GFX_HACK_MISSING_COLOR_VALUE{{System::GFX, "Hacks", "MissingColorValue"}, 0xFF};
```

Comment:
For future flexibility, it might be wiser to allow the game ini to specify a 4 channel RGBA color. 

Because whatever is happening in underlying hardware, I really doubt it replicates the same value across all four channels.

Reply:
Done (and tested by confirming that `MissingColorValue = 0xFF00FFFF` results in magenta debug cubes).